skipper.go

```diff
@@ -1200,6 +1259,32 @@ func run(o Options, sig chan os.Signal, idleConnsCH chan struct{}) error {
 		)
 	}
 
+	oauthConfig := &auth.OAuthConfig{}
+	if o.EnableOAuth2GrantFlow /* explicitly enable grant flow */ {
+		oauthConfig.AuthURL = o.OAuth2AuthURL
+		oauthConfig.TokenURL = o.OAuth2TokenURL
+		oauthConfig.TokeninfoURL = o.OAuthTokeninfoURL
+		oauthConfig.SecretFile = o.OAuth2SecretFile
+		oauthConfig.ClientID = o.OAuth2ClientID
+		oauthConfig.ClientSecret = o.OAuth2ClientSecret
+		oauthConfig.ClientIDFile = o.OAuth2ClientIDFile
+		oauthConfig.ClientSecretFile = o.OAuth2ClientSecretFile
+		oauthConfig.CallbackPath = o.OAuth2CallbackPath
+		oauthConfig.AuthURLParameters = o.OAuth2AuthURLParameters
+		oauthConfig.SecretsProvider = sp
+		oauthConfig.Secrets = secrets.NewRegistry()
+		oauthConfig.AccessTokenHeaderName = o.OAuth2AccessTokenHeaderName
+		oauthConfig.TokeninfoSubjectKey = o.OAuth2TokeninfoSubjectKey
+		oauthConfig.TokenCookieName = o.OAuth2TokenCookieName
+		oauthConfig.ConnectionTimeout = o.OAuthTokeninfoTimeout
+		oauthConfig.MaxIdleConnectionsPerHost = o.IdleConnectionsPerHost
+		oauthConfig.Tracer = tracer
+
+		if err := initGrant(oauthConfig, &o); err != nil {
+			log.Errorf("Error while initializing oauth grant filter: %v.", err)
```

Comment:
Error while initializing .. -> Failed to initialize ..

Reply:
Good point. Fixed.